# HG changeset patch
# User Jack Magne <jmagne@redhat.com>
# Date 1506640850 25200
#      Thu Sep 28 16:20:50 2017 -0700
# Node ID 252c10f448971b7ae087bde259505abd5dc5a03a
# Parent  3e9a5ae2149d04877dc19b117a8917c22854f8eb
Fix: Bug 1400884 - new JSS failures: HMAC Unwrap and KeyWrapping FIPSMODE.

diff --git a/org/mozilla/jss/pkcs11/KeyType.java b/org/mozilla/jss/pkcs11/KeyType.java
--- a/org/mozilla/jss/pkcs11/KeyType.java
+++ b/org/mozilla/jss/pkcs11/KeyType.java
@@ -204,9 +204,7 @@
                             EncryptionAlgorithm.AES_192_CBC,
                             EncryptionAlgorithm.AES_256_ECB,
                             EncryptionAlgorithm.AES_256_CBC,
-                            /* AES CBC PAD is the same as AES_256_CBC_PAD */
-                            /* shouldn't break backward compatibility 313798*/        
-                            //EncryptionAlgorithm.AES_CBC_PAD, 
+                            EncryptionAlgorithm.AES_CBC_PAD, 
                             EncryptionAlgorithm.AES_128_CBC_PAD,
                             EncryptionAlgorithm.AES_192_CBC_PAD,
                             EncryptionAlgorithm.AES_256_CBC_PAD        
